dali/sasha/sadbghk.cpp

```diff
@@ -35,12 +36,9 @@ class CSashaDebugPlaneHousekeepingServer : public ISashaServer, public Thread
 
     virtual void start() override
     {
-        expiryDays = getComponentConfigSP()->getPropInt("@expiryDays", defaultExpiryDays);
-        if (expiryDays == 0)
-        {
-            WARNLOG("Failed to get expiry days for debug plane");
-            return;
-        }
+        Owned<IPropertyTree> compConfig = getComponentConfig();
+        expiryDays = compConfig->getPropInt("@expiryDays", defaultExpiryDays);
+        interval = compConfig->getPropInt("@interval", defaultHouseKeepingIntervalHours);
```

Comment:
minor: cleaner to check these here and exit (return), and then unconditionally set stopped to false, just before thread starts.